sympy/solvers/ode.py

```diff
@@ -4049,8 +4049,8 @@ def ode_order_reducing_substitution(eq, func, order, match):
         if not eq.has(g(x)):
             break
     eq = eq.subs(f(x).diff(x, match['var']), g(x))
```

Comment:
Hi,
What is to be done now?Is my code gonna be added?I want to participate in gsoc this year.Atleast try with some good probability.

Reply:
```
>>> a =sum([j(x).diff(x,i) for i in range(5) for j in [f,g]])
>>> a
f(x) + g(x) + Derivative(f(x), x) + Derivative(f(x), (x, 2)) + Derivative(f(x), (x, 3)) + Derivative(f(x), (x, 4)) + Derivative(g(x), x) + Derivative(g(x), (x, 2)) + Derivative(g(x), (x, 3)) + Derivative(g(x), (x, 4))
>>> list(a.atoms(Function))
[g(x), f(x)]
>>>
```